src/vivarium_gates_iv_iron/constants/data_values.py

```diff
@@ -64,14 +78,26 @@ class _HemoglobinDistributionParameters(NamedTuple):
 IV_IRON_EFFECT_SIZE = (23, 14)  # (mean, sd) g/L
 
 # Risk Effects
-RR_MATERNAL_HEMORRHAGE_ATTRIBUTABLE_TO_HEMOGLOBIN = (3.54, 1.2, 10.4)  # (median, lower, upper) 95% CI
+RR_MATERNAL_HEMORRHAGE_ATTRIBUTABLE_TO_HEMOGLOBIN = (
+    3.54,
+    1.2,
+    10.4,
+)  # (median, lower, upper) 95% CI
 HEMOGLOBIN_SCALE_FACTOR_MODERATE_HEMORRHAGE = 0.9
 HEMOGLOBIN_SCALE_FACTOR_SEVERE_HEMORRHAGE = 0.833
 
 TMREL_HEMOGLOBIN_ON_MATERNAL_DISORDERS = 120.0
 RR_SCALAR = 10.0  # Conversion factor between hemoglobin units (g/L) and relative risk units (g/dL)
 
+RR_STILLBIRTH_PROBABILITY_ATTRIBUTABLE_TO_HEMOGLOBIN = (
+    3.87,
+    1.88,
+    8.06,
+)  # (median, lower, upper) 95% CI
+
 _IFA_EFFECT_SIZE_LOWER = 4.08
 _IFA_EFFECT_SIZE_UPPER = 11.52
-_IFA_EFFECT_SIZE_SD = (_IFA_EFFECT_SIZE_UPPER - _IFA_EFFECT_SIZE_LOWER) / (2 * 1.96)  # 95% CI
+_IFA_EFFECT_SIZE_SD = (_IFA_EFFECT_SIZE_UPPER - _IFA_EFFECT_SIZE_LOWER) / (
```

Comment:
I'm not a huge fan of how black handles mathematical expressions like this :/

Reply:
Agree. This is why I held off on the `loader.py` bits. Maybe Black isn't our answer.

Reply:
Why not just manually override this expression? (I agree it's ugly - kind of an edge case with a long numerator and short denominator)

Or is it kind of a group "policy" to follow black/pep-8 at all costs?

Reply:
We follow PEP-8, but our use of Black is something that recently was added. I'm not sure if there are things we want to change in our `pyproject.toml`.